packages/relay/src/ServerConfigParams.ts

```diff
@@ -21,6 +21,8 @@ export enum LoggingProviderMode {
   CHATTY
 }
 
+export const HUB_FILE = '/hub.lock'
```

Comment:
nitpic: better put the "slash" when joining the string. otherwise, it looks like the file is at the root..

Reply:
Ok